### DISCUSSION TITLE
Status of BindMapList

### DISCUSSION THREAD

**Discussion by PaulWaldo** [fyne-io/fyne#3193] [Q&A] [answered]
https://stackoverflow.com/a/68086932/1290460 says

> In v2.1 we plan to add binding.BindMapList which would be passed to NewTableWithData

I have looked through the branches and don't see any evidence of this feature.  Is it still planned and when might it be available?  Thanks!

**Comment by Jacalz** [comment, marked as answer]
Hmm. That’s a very good question. I can’t find anything here on GitHub when searching for `BindMapList`, which leads me to suspect that it was forgotten about and never materialised. The closest that I can find is https://github.com/fyne-io/fyne/issues/1853.

I suggest opening a feature request in case you want it added :)

**Comment by andydotxyz** [comment]
This was held up for `Table` because we could not agree if it would be a `MapList` or a `ListList`.
A ListList has the benefit of not needing extra mapping to get the data displayed, and matches the current usage.
However will a MapList better reflect future usage where columns are added and more...?

**Reply by PaulWaldo** [reply]
My vote would be for the most future flexibility (which is MapList if I understood your comment correctly)

**Comment by andydotxyz** [comment]
Most flexibility means more verbose APIs though !

**Reply by PaulWaldo** [reply]
Do you have any examples (pseudocode even) of each way of doing it @andydotxyz?  Maybe it could then be discussed with a wider audience.

**Reply by andydotxyz** [reply]
No work has begun on this yet
